coordinator/src/models/fhir/medication-dispense.ts

```diff
@@ -0,0 +1,35 @@
+import * as extension from "./extension"
+import * as common from "./common"
+import * as patient from "./patient"
+
+export interface MedicationDispense extends common.Resource {
+    identifier: Array<common.Identifier>
+    extension?: Array<extension.CodingExtension>
+    medicationCodeableConcept: common.CodeableConcept
+    subject: common.IdentifierReference<patient.Patient>
+    quantity: common.SimpleQuantity
+    authorizingPrescription: Array<AuthorizingPrescription>
+    whenPrepared: string
+    dosageInstruction: Array<DosageInstruction>
+    performer: Array<DispensePerformer>
+    type: common.CodeableConcept
+}
+
+export interface AuthorizingPrescription extends common.Resource {
```

Comment:
are these definitely FHIR resources? according to [the IG](https://simplifier.net/guide/DigitalMedicines/DM-MedicationDispense) an AuthorizingPrescription is a reference to a MedicationRequest (so i guess would be something like: `extends common.Reference<fhir.MedicationRequest>`

Reply:
it seems like since its referencing something outside of the Bundle so maybe it should be an identifier reference? not fully sure

Reply:
Couldn't find a model which matches this already, with identifier and extension properties, so if you know where it is please point me in right direction! Reference does not have these properties. I've removed the extends common.resource from these sub-models

Reply:
`common.IdentifierReference<fhir.MedicationRequest>` seems like the best fit for this? so this `AuthorizingPrescription` would extend that with an `extensionExtension` array

Reply:
I think I agree more with another point you made on a separate thread on this PR, that the interface should be separate for dispense vs prescribe, an authorizing prescription isn't a direct reference to a MedicationRequest